libcontainer/cgroups/fs/fs.go

```diff
@@ -129,14 +129,15 @@ func (m *Manager) Apply(pid int) (err error) {
 			// later by Set, which fails with a friendly error (see
 			// if path == "" in Set).
 			if isIgnorableError(c.Rootless, err) && c.Path == "" {
+				retErr = cgroups.ErrRootless
```

Comment:
Do you think this is too strict? Maybe we should add a condition here:
```
if name == "device" {
    retErr = cgroups.ErrRootless
}
```

Reply:
It doesn't really matter here because there can't be a situation where devices cgroup can't be created while others can.